apis/apps/v1alpha1/cluster_types.go

```diff
@@ -1651,8 +1651,11 @@ func (t *InstanceTemplate) GetName() string {
 	return t.Name
 }
 
-func (t *InstanceTemplate) GetReplicas() *int32 {
-	return t.Replicas
+func (t *InstanceTemplate) GetReplicas() int32 {
+	if t.Replicas != nil {
+		return *t.Replicas
+	}
+	return 1
```

Comment:
So 1 is the default replicas if it's not specified explicitly?

Reply:
yes

Reply:
then put 1 to a constant